cdconn00/clamify-ui
# [CLAM-6] Dockerize project.

## [CLAM-6]

---

### Summary

- Creates a docker file and related files to create a Docker container of the running application.

---

### Testing

1. Run all the unit tests.
2. Run all the integration tests.
3. Verify the image builds and runs.

---

### Post-Deployment Validation

N/A until CI/CD setup.

---


[CLAM-6]: https://clamify.atlassian.net/browse/CLAM-6?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
